src/citrine/builders/design_spaces.py

```diff
@@ -329,9 +329,8 @@ def migrate_enumerated_design_space(*,
 
     if cleanup:
         # archive the old enumerated design space
-        enumerated_ds.archived = True
         try:
-            project.design_spaces.update(enumerated_ds)
+            project.design_spaces.archive(enumerated_ds.uid)
```

Comment:
This change makes the method more robust and more idiomatic.  Its also easier to fake in the tests.